contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,382 @@
+
+In the below example, we merge the user with `alias_id` into the user with `distinct_id`. This means that the user with `distinct_id` is considered to be the original user, and `alias_id` is considered to be the alias. Any events submitted with `identify` using either `distinct_id` or `alias_id` will be associated with the same user.
+
+<MultiLanguage selector="tabs">
+
+```node
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'alias_id',
+})
+```
+
+```javascript
+posthog.alias('alias_id', 'distinct_id');
+```
+
+```python
+posthog.alias('alias_id', 'distinct_id');
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id", "alias_id");
+```
+
+</MultiLanguage>
+
+Note that is not possible not be to merge the same `alias_id` to two different distinct IDs. For example:
+
+<MultiLanguage selector="tabs">
+
+```node
+client.alias({
+    distinctId: 'distinct_id_one',
+    alias: 'alias_id',
+})
+// ❌ Not possible
+client.alias({
+    distinctId: 'distinct_id_two',
+    alias: 'alias_id',
+})
+```
+
+```javascript
+posthog.alias('alias_id', 'distinct_id_one');
+// ❌ Not possible
+posthog.alias('alias_id', 'distinct_id_two');
+
+```
+
+```python
+posthog.alias('alias_id', 'distinct_id_one');
+# ❌ Not possible
+posthog.alias('alias_id', 'distinct_id_two');
+
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id_one',
+  'alias' => 'alias_id'
+));
+// ❌ Not possible
+PostHog::alias(array(
+  'distinctId' => 'distinct_id_two',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id_one",
+  alias: "alias_id",
+})
+# ❌ Not possible
+posthog.alias({
+  distinct_id: "distinct_id_two",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id_one",
+  Alias: "alias_id",
+})
+// ❌ Not possible
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id_two",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id_one", "alias_id");
+// ❌ Not possible
+posthog.alias("distinct_id_two", "alias_id");
+```
+
+</MultiLanguage>
+
+<!-- ### Merging dangerously (TBD)
+
+1. `merge`, with no safeguards. We don't recommend using this in your code, but rather as a one-off manually for recovering from implementation problems. 
+   
+> **Warning:** Merging users with `merge` is irreversible! Be careful not to merge users who should not be merged together.
+
+For example using the posthog python library:
+
+```py
+posthog.alias('user-id', 'non-identified-id')
+
+posthog.capture('user-id', '$merge_dangerously', {'alias': 'second-user-id'})
+
+``` -->
```

Comment:
I dont fully understand merge yet - would love to discuss on a huddle, and if it's even worth including. 

Also, I didn't find a `merge` function in the Node SDK. Is it only possible to call with `capture`?